Src/CSharpier/ConfigurationOptions.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace CSharpier
+{
+    public class ConfigurationOptions
+    {
+        public HashSet<string> Exclude { get; set; }
```

Comment:
If configuration options only have `Exclude`, why not just have a `.csharpierignore` file? 

Then like `.gitignore`s, there can be multiple in a project, and scoped down to a given directory.

Reply:
Good call. This was just my quick hack so I could get the format repos workflow passing. I updated https://github.com/belav/csharpier/issues/43 with the idea